business-development.md

```diff
@@ -4,32 +4,28 @@ Our team is comprised entirely of developers, which makes the business developme
 
-
 ### How Our Developers are Different
 
 Base Two was founded by people who had a lot of experience in other consulting firms. They saw strengths and weaknesses in the traditional model, and wanted to try something else. Our focus, instead of being sales-oriented, is on people. We feel the most valuable resource in the tech market is having the best developers and keeping them happy. We hire people who have an aptitude for learning and a passion for technology, and we create a structure that allows them to do their best work.
 
 Developers at Base Two are not one-dimensional. In addition to being excellent at their craft, they have:
+
 - Insight into the business
 - Empathy for the problem they're trying to solve
 - Innate curiosity and enthusiasm that will drive them to seek out the best solutions
 
-
 ### How Our Sales Process Works
 
-We are not a sales-oriented company. We're not out there every day trying to find the next gig. In order to stay in business, our strategy is to develop *sustainable* relationships with our clients. We make sure we're always providing value to our clients since we don't have a huge pool of leads. We're flexible at the beginning of a relationship as we figure out how best to work together with our clients, and we ask our clients to be flexible through the evolving process.
-
+We are not a sales-oriented company. We're not out there every day trying to find the next gig. In order to stay in business, our strategy is to develop _sustainable_ relationships with our clients. We make sure we're always providing value to our clients since we don't have a huge pool of leads. We're flexible at the beginning of a relationship as we figure out how best to work together with our clients, and we ask our clients to be flexible through the evolving process.
 
 ### Who We Work With
 
-Our roster runs the gamut from small, bootstrap startups to fortune 500 companies and everything in between. It may sound crazy for such a small team but we have 3-4 billion-dollar clients that we work with, and we also have 3-4 shoestring clients we work with. We're flexible, and we're careful with our clients' money.
-
+Our roster ranges from small, bootstrap startups to fortune 500 companies and everything in between. It may sound crazy for such a small team but we have 3-4 billion-dollar clients that we work with, and we also have 3-4 shoestring clients we work with. We're flexible, and we're careful with our clients' money.
 
 ## Conversation End Goal
 
-Flip the conversation (get on the "same side of the table") to collaboratively solve a problem. Even if we're not the right return-on-the-dollar for them, how can we be helpful? Can we recommend a contractor, can we recommend other companies?
+We strive to collaboratively solve each problem with our client. Even if we're not the right return-on-the-dollar for them, how can we be helpful? Can we recommend a contractor, can we recommend other companies?
```

Comment:
The notion of "getting on the same side of the table" as them has come up as an explicit conversational strategy in the past. I'm not sure that "strive to collaboratively solve" is as useful as an end-goal strategy. Maybe we should reword the original idea to be more direct. This is probably too wordy, so anyone please help me pare it down:

> We're most successful with clients when we're able to transition from being seen as a vendor trying to sell a solution, to instead them feeling that we're hearing and understanding their problem, brainstorming possible solutions. We think of this as trying to get on the same side of the table as the client, collaborating on their issue. We want them to feel that we're good listeners for any problems they might have, not only those that they believe are explicitly in our wheelhouse. The contracts and sales tend to flow naturally out of a relationship where they first and foremost feel comfortable brainstorming or just venting about whatever problems might be on their plate.

Reply:
I think this is perfect, thank you!!